app/views/dishes/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1><%= @dish.name %> </h1>
+<p><%= @dish.description %></p>
+
+<h3>Ingredients:</h3>
+<ul>
+    <% @dish.ingredients.each do |ingredient| %>
+        <li><%= ingredient.name %> = <%= ingredient.calories %> calories</li>
+    <% end %>
+</ul>
+
+<p>Total Calories: <%= @dish.total_calories %></p>
+
+<p>Chef: <%= @dish.chef.name %></p>
+
+<h3>AddIngredient:</h3>
+<%= form_with url: add_ingredient_dish_path(@dish), method: :post, local: true do |form| %>
+    <%= form.label :ingredient_id, "Ingredient ID" %>
+    <%= form.number_field :ingredient_id %>
+    <%= form.submit "Add Ingredient" %>
+<% end %>
```

Comment:
Everything in this view is great except for the path for the form. I already explained the correct path in the routes.rb file.